main.py

```diff
@@ -0,0 +1,53 @@
+import telebot
+import pyqrcode
+import config
+import requests
+from PIL import Image
+from pyzbar.pyzbar import decode
+from io import BytesIO
+
+bot = telebot.TeleBot(config.TOKEN)
```

Comment:
Move token to [environment variables](https://en.wikipedia.org/wiki/Environment_variable#:~:text=An%20environment%20variable%20is%20a,in%20which%20a%20process%20runs.)

[How to Set and Get Environment Variables in Python](https://able.bio/rhett/how-to-set-and-get-environment-variables-in-python--274rgt5)